pyiron/atomistics/job/structurecontainer.py

```diff
@@ -2,62 +2,160 @@
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
 
-from __future__ import print_function
-from pyiron.atomistics.structure.atoms import Atoms
-from pyiron.atomistics.job.atomistic import AtomisticGenericJob
-
-__author__ = "Yury Lysogorskiy"
+__author__ = "Yury Lysogorskiy, Jan Janssen, Marvin Poul"
 __copyright__ = (
     "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
     "Computational Materials Design (CM) Department"
 )
-__version__ = "1.0"
-__maintainer__ = "Jan Janssen"
-__email__ = "janssen@mpie.de"
+__version__ = "0.1"
+__maintainer__ = "Marvin Poul"
+__email__ = "poul@mpie.de"
 __status__ = "development"
-__date__ = "Sep 1, 2017"
+__date__ = "Aug 12, 2020"
 
+from pyiron.base.generic.inputlist import InputList
+from pyiron.atomistics.job.atomistic import AtomisticGenericJob
+from pyiron.atomistics.structure.atoms import Atoms
+
+class StructureList(AtomisticGenericJob):
+    """
+    Container to save a list of structures in HDF5 together with tags.
+
+    Add new structures with :meth:`.StructureList.append`, they are
+    added to :attr:`.StructureList.structures`.  The HDF5 is written when
+    :meth:`.run` is called.
+    """
+
+    __version__ = "0.1.0"
 
-class StructureContainer(AtomisticGenericJob):
     def __init__(self, project, job_name):
-        super(StructureContainer, self).__init__(project, job_name)
-        self.__version__ = "0.1"
-        self.__name__ = "StructureContainer"
+        super().__init__(project, job_name)
+        self.__structures = InputList(table_name = "structures")
         self.server.run_mode.interactive = True
 
     @property
-    def structure(self):
-        return self._structure
+    def structures(self):
+        """
+        :class:`.InputList`: list of structures
+        with meta data; each item in this list has an 'atoms' key for the
+        atomic structures and then as many additional keys as passed to
+        :meth:`~.StructureContainer.append()`
+        """
+        return self.__structures
 
-    @structure.setter
-    def structure(self, structure):
-        if isinstance(structure, AtomisticGenericJob):
+    @staticmethod
+    def __to_structure(structure_or_job):
+        """
```

Comment:
Single _ or two __ ? Currently in pyiron we use just a single one.

Reply:
Ah, just old habit, will change.

Reply:
@pmrv Can you fix this?

Reply:
Didn't see this, it's fixed now.